cmd/soroban-cli/src/utils/contract-workspace-template/Cargo.toml.removeextension

```diff
@@ -5,7 +5,7 @@ members = [
 ]
 
 [workspace.dependencies]
-soroban-sdk = "21.0.0"
+soroban-sdk = "22.0.0-rc.3"
```

Comment:
The CLI should probably only ever generate templates for stable releases, not release candidates. Even if folks are using an RC of the CLI it's unclear to me it should encourages folks to adopt other unstable software releases in other areas. Can we keep this as the stable?
```suggestion
soroban-sdk = "22.0.0"
```

Reply:
I think this check was more for our development process, but if we are fine with tests temporarily failing in dev once we bump the version, then I agree we should make it always stable

Reply:
Why would the test fail? I'm suggesting that the init function should generate from a template that is always the latest stable, so that might mean that 22-rc should be initing with an sdk that is 21.

Reply:
Oh I see what you mean. Previously test was failing because cli version is 22-rc and test checks that the template uses same major version. Should we keep that condition (test will fail on rc though) or maybe allow for -rc to use previous stable version?